DeviceManager/DeviceManager/View/OnBoardingVC.swift

```diff
@@ -8,9 +8,41 @@
 
 import UIKit
 
-class ViewController: BaseVC {
-
+class OnBoardingVC: BaseVC {
+    @IBOutlet weak var onClick: UIButton!
+    @IBOutlet weak var pageControl: UIPageControl!
+    let model =  WalkThroughVM()
     override func viewDidLoad() {
         super.viewDidLoad()
+        model.onBoarding()
+        onClick.layer.borderWidth = 2
+        onClick.layer.borderColor = UIColor.black.cgColor
+    }
+}
+
+extension OnBoardingVC: UICollectionViewDataSource, UICollectionViewDelegate {
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return model.info.count
+    }
+    func scrollViewDidEndDecelerating(_ scrollView: UIScrollView) {
+        let pageNumber = round(scrollView.contentOffset.x / scrollView.frame.size.width)
+        pageControl.currentPage = Int(pageNumber)
+        print(scrollView.contentOffset.x)
+    }
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let cell = collectionView.dequeueReusableCell(withReuseIdentifier: String(describing: "OnBoardingCVC"), for: indexPath as IndexPath) as? OnBoardingCVC
+        cell?.imageOutlet.image =  UIImage(named: model.info[indexPath.row].images)
```

Comment:
You can directly save image instead of just String type 
`cell?.imageOutlet.image =  model.info[indexPath.row].image` 
where your model in VM will have `UIImage(named: String)` code